docs/_posts/murat-gunay/2021-12-29-classifierdl_urduvec_fakenews_ur.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: model
+title: Fake News Classifier in Urdu
+author: John Snow Labs
+name: classifierdl_urduvec_fakenews
+date: 2021-12-29
+tags: [urdu, fake_news, fake, ur, open_source]
+task: Text Classification
+language: ur
+edition: Spark NLP 3.3.1
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model determines if news articles in Urdu are real or fake.
+
+## Predicted Entities
+
+`real`, `fake`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/classifierdl_urduvec_fakenews_ur_3.3.1_3.0_1640771335815.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler() \
+      .setInputCol("news") \
+      .setOutputCol("document")
+
+tokenizer = Tokenizer() \
+      .setInputCols(["document"]) \
+      .setOutputCol("token")
+    
+normalizer = Normalizer() \
+      .setInputCols(["token"]) \
+      .setOutputCol("normalized")
+
+lemma = LemmatizerModel.pretrained("lemma", "ur") \
+         .setInputCols(["normalized"]) \
+         .setOutputCol("lemma")
+
+embeddings = WordEmbeddingsModel.pretrained("urduvec_140M_300d", "ur") \
+      .setInputCols("document", "lemma") \
+      .setOutputCol("embeddings")
+
+embeddingsSentence = SentenceEmbeddings() \
+      .setInputCols(["document", "embeddings"]) \
+      .setOutputCol("sentence_embeddings") \
+      .setPoolingStrategy("AVERAGE")
+
+classifierdl = ClassifierDLModel.pretrained("classifierdl_urduvec_fakenews", "ur") \
+  .setInputCols(["document", "sentence_embeddings"]) \
+  .setOutputCol("class")
+
+urdu_fake_pipeline = Pipeline(stages=[document_assembler, tokenizer, normalizer, lemma, embeddings, embeddingsSentence, classifierdl])
+
+light_pipeline = LightPipeline(urdu_fake_pipeline.fit(spark.createDataFrame([['']]).toDF("news")))
+
+result = light_pipeline.annotate("پاکستان کے اخبارات روزانہ اس بات کی مثالیں دیتے ہیں کہ کس طرح مصنوعی ذہانت کو ذاتی مقاصد کے لیے استعمال کیا جا سکتا ہے۔")
```

Comment:
Two things:
1) Is this a real text? I can't find it in the Internet. What is the source? If it's a translation, use real examples from Wikipedia.
2) Why is this fake? 
![image](https://user-images.githubusercontent.com/36634572/147751533-46f578e5-9171-474f-99e2-7809a04dd6cf.png)

Reply:
@josejuanmartinez 
I took it from the train data-set. 
There can't be a county whose all newspapers give daily news regarding AI for personal purposes. I don't see any news regarding AI uses for personal purposes. Besides, it happens daily.... everyday... impossible, I guess.